WORKSPACE

```diff
@@ -0,0 +1,35 @@
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+
+git_repository(
+    name = "build_bazel_rules_apple",
+    remote = "https://github.com/bazelbuild/rules_apple.git",
+    tag = "0.14.0",
+)
+
+load(
+    "@build_bazel_rules_apple//apple:repositories.bzl",
+    "apple_rules_dependencies",
+)
+
+apple_rules_dependencies()
+
+load(
+    "@build_bazel_rules_swift//swift:repositories.bzl",
+    "swift_rules_dependencies",
+)
+
+swift_rules_dependencies()
+
+load(
+    "@build_bazel_apple_support//lib:repositories.bzl",
+    "apple_support_dependencies",
+)
+
+apple_support_dependencies()
+
+# For packaging python scripts.
+git_repository(
+    name = "subpar",
+    remote = "https://github.com/google/subpar",
+    commit = "edb8409041b521959b6ed4b7412c0eec59d4af78",
```

Comment:
How do we deal with this in PCX w/o internet connection?

Reply:
We have ways ;-)